File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.cc
#include "iostream"
#include "string"

using namespace std;

int main() {
    int n;
    cin >> n;
    int cnt = 0;
    int num = 666;
    string target = "666";
    while (true) {
        if (to_string(num).find(target) != string::npos) {
            cnt++;
        }
        if (cnt == n) {
            cout << num << "\n";
            break;
        }
        num++;
    }
}

File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine());
        String six = "666";
        int num = 666;
        int cnt = 1;
        while (cnt != N) {
            num++;
            if (Integer.toString(num).contains(six)) {
                cnt++;
            }
        }
        System.out.println(num);
    }
}
